.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -1 +1,5 @@
-blank_issues_enabled: false
\ No newline at end of file
+blank_issues_enabled: false
+contact_links:
+  - name: 🚗 Bug in CARLA Simulator
+    url: https://github.com/carla-simulator/carla/issues/new
+    about: Found a bug in the CARLA simulator?
```

Comment:
Probably the contact links are a copy paste mistake?

Reply:
Not really, it was intentionally to make it possible to report a bug in Carla itself. Basically this adds a third option in the wizard to directly create an issue in the Carla repo.

However it might be superflous.